Simulator/Vertices/NG911/All911Vertices.cpp

```diff
@@ -65,7 +70,7 @@ void All911Vertices::createAllVertices(Layout &layout)
       assert(*vi < size_);
```

Comment:
Is this code CPU-only (for example, used only for loading graphs during initialization) or is it something that would also be implemented in the future on the GPU? If the former case, then this is fine; no change needed. If the latter case, then we want to avoid libraries not available on the GPU and we want to use indices into vectors, rather than vertex pointers. I suspect the former, but I want to double-check.

Reply:
Yes, this is CPU-only code executed during initialization for loading the graph into Graphitti's data structures.